packages/web-components/@sfx/base/src/base.ts

```diff
@@ -0,0 +1,72 @@
+import { LitElement, customElement, html } from 'lit-element';
+import { makeSlot, createChildrenObserver } from './utils';
+
+/**
+ * Extends LitElement and all components extend it.
+ */
+@customElement('sfx-base')
+export default class Base extends LitElement {
+  /**
+   * MutationObserver created via createChildrenObserver function.
+   * See [[createChildrenObserver]] for details. This function was
+   * pulled directly from
+   * github.com/Polymer/lit-element/issues/42#issuecomment-442894676.
+   */
+  observer: MutationObserver;
+
+  /**
+   * Constructs an instance of the Base class and calls the addSlots
```

Comment:
I would link `addSlots`. Doing `[[addSlots]]` may be sufficient, but double-check with the generated output.